mapper/orm-coordination-database-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/databasepolling/impl/OutboxEventBackgroundProcessor.java

```diff
@@ -107,7 +110,7 @@ public DatabasePollingOutboxWorker(SessionFactoryImplementor sessionFactory) {
 			}
 
 			try ( SessionImplementor session = (SessionImplementor) mapping.sessionFactory().openSession() ) {
-				transactionHelper.begin( session, null );
+				transactionHelper.begin( session, transactionTimeout );
```

Comment:
> This could do with a test in JTA mode? I think we have some already, we can just copy/paste them and adapt them.
> 
> You could some custom ORM listener that introduces an artificial delay which would make automatic indexing fail with the default transaction timeout, but succeed with the custom one configured though your new configuration property. E.g. something similar to `org.hibernate.search.util.impl.integrationtest.mapper.orm.TimeoutLoadingListener`, but one that would always be registered (you can use `hibernate.session.events.auto` for that).
> 
> Be sure to test both cases, as simply having automatic indexing succeed doesn't prove anything unless you know it would have failed without your setting.

I don't know if we have already without Spring...